modules/monitoring/library/Monitoring/Command/Object/AddCommentCommand.php

```diff
@@ -46,4 +53,27 @@ public function getPersistent()
     {
         return $this->persistent;
     }
+
+    /**
+     * Set the time when the acknowledgement should expire
+     *
+     * @param   int $expireTime
+     *
+     * @return  $this
+     */
+    public function setExpireTime($expireTime)
+    {
+        $this->expireTime = (int) $expireTime;
+        return $this;
```

Comment:
Please add an empty line before the return statement. We require empty lines before return statements.